tensorbay/cli/log.py

```diff
@@ -314,26 +338,33 @@ def _add_graph_full(
     ) -> str:
         log = _get_full_log(commit, branch_names)
         splitlines = iter(log.splitlines())
-        prefixes = ["|"] * self._layer
+        prefixes = self._get_colorful_prefix()
         # Don't merge branches.
         if self._merge_pointer is None:
             details_prefix = " ".join(prefixes)
             lines = [f"{self._get_title_prefix(prefixes, original_pointer)} {next(splitlines)}\n"]
             lines.extend(f"{details_prefix} {line}\n" for line in splitlines)
         else:
             # Merge branches.
-            prefixes.append("|")
+            prefixes.append(click.style("|", fg=self._graph_colors[self._merge_pointer]))
+            del self._graph_colors[self._merge_pointer]
             merge_prefixes = self._get_merge_prefixes()
             lines = [f"{self._get_title_prefix(prefixes, original_pointer)} {next(splitlines)}\n"]
             lines.extend(self._combine_details(list(merge_prefixes), list(splitlines)))
-
         return "".join(lines)
 
+    def _get_colorful_prefix(self) -> List[str]:
+        return [click.style("|", fg=self._graph_colors[i]) for i in range(self._layer)]
```

Comment:
`return [click.style("|", fg=color) for color in islice(self._layer_colors, self._layer)]`